src/towncrier/_builder.py

```diff
@@ -129,8 +140,10 @@ def find_fragments(
         file_content = {}
 
         for basename in files:
+            if basename == template_filename:
+                continue
             issue, category, counter = parse_newfragment_basename(
-                basename, config.types
+                basename, config.types, strict
             )
```

Comment:
Maybe we can have something like this 
```suggestion
                basename, config.types
            )
        if strict and not (issue and category):
            raise ClickException(f'Invalid news fragment found: {basename}.')
```